common/management/src/lib.rs

```diff
@@ -13,12 +13,15 @@
 // limitations under the License.
 
 mod cluster;
+mod role;
 mod stage;
 mod udf;
 mod user;
 
 pub use cluster::ClusterApi;
 pub use cluster::ClusterMgr;
+pub use role::role_api::RoleMgrApi;
```

Comment:
pub use role::RoleMgrApi; -- short use is always better

Reply:
thank you, fixed